solidity/test/bridge/Bridge.test.ts

```diff
@@ -892,30 +892,52 @@ describe("Bridge", () => {
 
                       it("should update the depositors balances", async () => {
                         // The sum of sweep tx inputs is 4148000 satoshi. The output
-                        // value is 4145001 so the fee is 2999. There is 5 deposits
-                        // so 599 satoshi fee should be incurred per deposit.
-                        // Each deposit should also incur the treasury fee whose
-                        // initial value is 0.05% of the deposited amount.
+                        // value is 4145001 so the sweep transaction fee is 2999.
```

Comment:
Clear example that `depositTxMaxFee = 1000; // 1000 satoshi` in the constructor does not make sense 😬 

Not in this PR, but can we increase it to 100k? I know there is a TODO to revisit these values but 1000 is definitely not enough.

Reply:
Yeah this was just an initial value that will be revisited anyway. Also remember `depositTxMaxFee` is max transaction fee per deposit. Not for the entire sweep transaction.